internal/service/autoscaling/policy.go

```diff
@@ -637,10 +908,13 @@ func expandPredictiveScalingMetricSpecifications(metricSpecificationsSlice []int
 	}
 	metricSpecificationsFlat := metricSpecificationsSlice[0].(map[string]interface{})
 	metricSpecification := &autoscaling.PredictiveScalingMetricSpecification{
-		PredefinedLoadMetricSpecification:    expandPredefinedLoadMetricSpecification(metricSpecificationsFlat["predefined_load_metric_specification"].([]interface{})),
-		PredefinedMetricPairSpecification:    expandPredefinedMetricPairSpecification(metricSpecificationsFlat["predefined_metric_pair_specification"].([]interface{})),
-		PredefinedScalingMetricSpecification: expandPredefinedScalingMetricSpecification(metricSpecificationsFlat["predefined_scaling_metric_specification"].([]interface{})),
-		TargetValue:                          aws.Float64(float64(metricSpecificationsFlat["target_value"].(int))),
+		PredefinedLoadMetricSpecification:     expandPredefinedLoadMetricSpecification(metricSpecificationsFlat["predefined_load_metric_specification"].([]interface{})),
+		PredefinedMetricPairSpecification:     expandPredefinedMetricPairSpecification(metricSpecificationsFlat["predefined_metric_pair_specification"].([]interface{})),
+		PredefinedScalingMetricSpecification:  expandPredefinedScalingMetricSpecification(metricSpecificationsFlat["predefined_scaling_metric_specification"].([]interface{})),
+		CustomizedScalingMetricSpecification:  expandCustomizedScalingMetricSpecification(metricSpecificationsFlat["customized_scaling_metric_specification"].([]interface{})),
+		CustomizedLoadMetricSpecification:     expandCustomizedLoadMetricSpecification(metricSpecificationsFlat["customized_load_metric_specification"].([]interface{})),
+		CustomizedCapacityMetricSpecification: expandCustomizedCapacityMetricSpecification(metricSpecificationsFlat["customized_capacity_metric_specification"].([]interface{})),
+		TargetValue:                           aws.Float64(float64(metricSpecificationsFlat["target_value"].(int))),
```

Comment:
Small styling nit: these should be sorted alphabetically similar to the AWS Documentation at https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_PredictiveScalingMetricSpecification.html. It will make it easier to read if the documentation is pulled up side by side with the code